src/Sentry.Unity.Editor/Native/BuildPostProcess.cs

```diff
@@ -186,22 +186,25 @@ private static void UploadDebugSymbols(IDiagnosticLogger logger, BuildTarget tar
 
             DataReceivedEventHandler logForwarder = (object sender, DataReceivedEventArgs e) =>
             {
-                var msg = e.Data.ToString().Trim();
-                var msgLower = msg.ToLowerInvariant();
-                var level = SentryLevel.Info;
-                if (msgLower.StartsWith("error"))
+                if (!string.IsNullOrEmpty(e.Data))
```

Comment:
Interestingly enough: I noticed NullReferenceExceptions when attaching the debugger while building. Otherwise, Unity will just silently swallow those.